app/views/centers/index.html.erb

```diff
@@ -4,7 +4,7 @@
 <% @centers.each do |center| %>
   <tr>
     <td id="center_name"><%= center.name %></td>
-    <td id="details"><%= link_to 'Details', center %></td>
+    <td id="details"><%= link_to 'Details', center_path %></td>
```

Comment:
the center_path wouldn't work here. it should be `center_path(center)` as the path helper needs to know which record you want to go to